### PR TITLE
test: add tests for Avatar component

### LINKED ISSUE #539: Build Avatar component
### Link task to story

Story: https://github.com/Open-Study-College/osc/issues/538

### Description

n/a

### PR DESCRIPTION
**Pull request by mattwendzina**
Closes #539 

## 📝 Description

Adds tests for Avatar component

## ⛳️ Current behavior (updates)

No testing

## 🚀 New behavior

Adds tests

### CODE REVIEW

File: packages/osc-ui/src/components/Avatar/Avatar.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { Avatar } from './Avatar';
+
+const DELAY = 300;
+const IMAGE_URL =
+    'https://images.unsplash.com/photo-1492633423870-43d1cd2775eb?&w=128&h=128&dpr=2&q=80';
+const NAME = 'Colm Tuite';
+
+describe('Avatar component', () => {
+    const orignalGlobalImage = window.Image;
+    const initial = NAME.charAt(0);
```
Comment (stevenbister) [inline packages/osc-ui/src/components/Avatar/Avatar.spec.tsx:12]: Ha this is way better than I did it in the component itself 😂 Completely forgot about `charAt()`

File: packages/osc-ui/src/components/Avatar/Avatar.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { Avatar } from './Avatar';
+
+const DELAY = 300;
+const IMAGE_URL =
+    'https://images.unsplash.com/photo-1492633423870-43d1cd2775eb?&w=128&h=128&dpr=2&q=80';
+const NAME = 'Colm Tuite';
+
+describe('Avatar component', () => {
```
Comment (stevenbister) [inline packages/osc-ui/src/components/Avatar/Avatar.spec.tsx:10]: Do we need the describe block if we're grouping everything into it?

Comment (mattwendzina) [reply packages/osc-ui/src/components/Avatar/Avatar.spec.tsx:10]: I've tended to always use a describe block to group all tests for a given component, just from the conventions in the way I learnt, but not too fussed. If there were other specific groups I would wrap them in another describe, but I'm easy either way.

Comment (stevenbister) [reply packages/osc-ui/src/components/Avatar/Avatar.spec.tsx:10]: Yeah not a big deal, just curious. I've tended not to unless I'm grouping tests as the filename is included in the test runs e.g.

image

But yeah no biggie

**Review by stevenbister — APPROVED** [review]
LGTM!

**Review by hi-im-si — CHANGES_REQUESTED** [review]

**Comment by mattwendzina** [comment]
Now displays full initials @hi-im-si 👍

**Review by hi-im-si — APPROVED** [review]
LGTM 🚀

**Pull request merged**